### ISSUE TITLE
Bug Fix

### ISSUE THREAD

**Issue by http-ph** [ZerioDev/Music-bot#55]
Command: Clear Queue
If you have less than one song in the queue, and executing the command, it will generate an error, to fix it just add the following line:

```
let trackList = queue.tracks.length;
if(trackList <= 1) return message.channel.send(`${client.emotes.error} - There is only one song in the queue.`);
```
Need to add before the clear queue function.

Greetings.

**Comment by ZerioDev** [comment]
Hello 👋
Thank you for your participation, i will check this and if everything is correct modify it !

**Comment by ZerioDev** [comment]
Fixed in version 5.0.0 ✔️
